pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1981,6 +1981,16 @@ public boolean hasRealtimeTable(String tableName) {
     return hasTable(TableNameBuilder.REALTIME.tableNameWithType(tableName));
   }
 
+  /**
+   * Gets the current state of the table
```

Comment:
current state" has a different meaning in helix/pinot. I would suggest saying something about it being enabled ot not.